src/kirby/components/card/card.component.tns.ts

```diff
@@ -25,6 +16,7 @@ export class CardComponent extends ContentView implements OnInit {
   @Input() title: string;
   @Input() subtitle: string;
   @Input() themeColor?: ThemeColor;
+  @Input() elevation?: 'highlighted' | 'disabled';
```

Comment:
Not sure that's what we settled on? The GH issue says there should be an `isHighlighted` boolean property...
```suggestion
  @Input() isHighlighted = false;
  elevation = 'z2';
```